lib/compare_task.rb

```diff
@@ -94,9 +97,13 @@ def render_comparison(description1, description2, scopes, options = {})
         identical = false if description1[scope] || description2[scope]
       end
     end
-
     output = "Compared descriptions are identical.\n" + output if identical && common_scopes
 
+    if !identical_scopes.empty?
+    output += "Following #{Machinery::pluralize(identical_scopes.count, "scope is", "scopes are")} " \
```

Comment:
Line is too long. [102/100]<br>Use 2 (not 0) spaces for indentation.<br>Prefer single-quoted strings inside interpolations.